lib/android.dart

```diff
@@ -338,6 +347,20 @@ Future<void> _updateStylesFile({
         value: '#' + android12BackgroundColor);
   }
 
+  if (android12BrandingImagePath != null) {
+    replaceElement(
+        launchTheme: launchTheme,
+        name: 'android:windowSplashScreenBrandingImage',
+        value: '@drawable/branding');
+  }
+
+  if (android12BrandingImageDarkPath != null) {
```

Comment:
Handling a dark path is not necessary here.  Instead, pass dark image path in one of the calls to `_applyStylesXml`, which will flow to `android12BrandingImagePath` here.

Reply:
Also updated this one as well. Thank you.